core_backend/tests/api/test_dashboard.py

```diff
@@ -696,9 +699,28 @@ async def test_top_content(
             assert top_content[i].positive_votes == c["positive_votes"]
             assert top_content[i].negative_votes == c["negative_votes"]
 
+    async def test_top_content_archived(
+        self, content_data: pytest.FixtureRequest, asession: AsyncSession
+    ) -> None:
+        """This test checks that the title of archived content is returned with the
+        string "[DELETED] " prepended to it. This string should be added to the
+        original content title and is displayed in the "Most Sent Content" table in the
+        Admin app dashboard to inform the user that the content has been removed.
+        """
+
+        top_n = 4
+        top_content = await get_top_content(
+            user_id=1, asession=asession, top_n=top_n, exclude_archived=True
```

Comment:
If `exclude_archived` is true, there should no content with `[DELETED] ` at the front.